openbb_platform/core/openbb_core/app/provider_interface.py

```diff
@@ -253,14 +253,22 @@ def _create_field(
         annotation = field.annotation
 
         additional_description = ""
+        choices: Dict = {}
```

Comment:
this variable name is misleading as it doesn't take only the choices but the extra info (being it choices and multiple items allowed info)

Reply:
This is a local variable that for iterating over choices for each the provider.  The only thing ever assigned to it are choices.

Reply:
is it? line 262 is `choices[p] = {"multiple_items_allowed": True, "choices": v.get("choices")}` 
sry if i'm misinterpreting

Reply:
Yes, this is a dictionary of choices for the field, by provider, where "multiple_items_allowed" is part of "choices".

Reply:
I also find this variable name ambiguous here because `choices` is also used with a different meaning as a key:value pair inside the dictionary